tests/acceptance/module/finance/manager/TariffPlansGroupingCest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace advancedhosters\hipanel\tests\acceptance\module\finance\manager;
+
+use hipanel\modules\finance\tests\_support\Page\plan\CreateGrouping as Create;
+use hipanel\modules\finance\tests\_support\Page\plan\View;
+use hipanel\modules\finance\tests\_support\Page\plan\UpdateGrouping as Update;
+use hipanel\modules\finance\tests\_support\Page\plan\Delete;
+use hipanel\modules\finance\tests\_support\Page\plan\Index;
+use hipanel\tests\_support\Step\Acceptance\Manager;
+
+class TariffPlansGroupingCest
```

Comment:
```suggestion
class GroupingTariffPlansCest
```

Ты тестируешь не группировку тарифных планов, а группирующие тарифные планы. Порядок слов другой

Reply:
Done